### ISSUE TITLE
Graphene multiscale implementation

### ISSUE THREAD

**Issue by akhileshh** [google/neuroglancer#461]
Hi Jeremy,

I have been working on implementing the multiscale mesh layer for graphene. I think I have most of it in place (#460).

However meshes seem to disappear if zoomed out or zoomed in too much. See attached video:

https://user-images.githubusercontent.com/10362782/235819672-9b2547f9-c8dd-4de8-8e7a-08f578606196.mp4

Graphene manifest is generated following the octree specification [here](https://github.com/google/neuroglancer/blob/master/src/neuroglancer/mesh/multiscale.ts#L67).

However, a chunkedgraph octree node may contain more than 8 children and something we call "skip connections" - `parent_layer - child_layer >= 1` instead of `parent_layer - child_layer == 1` in a standard octree representation. I think this is not a problem (?) because we can still see the meshes.

Another thing to note is that graphene multiscale meshes are generated by simplifying the finest meshes at each layer. They are not generated by meshing downsampled segmentation. So technically we are trying to use the same meshes for both legacy and multiscale graphene mesh layers.

Let me know if you need anything else to look into this. Myself or @sdorkenw can provide additional details.

Thanks!

**Comment by jbms** [comment]
Can you provide an example link (that works with your PR)?

Feel free to email directly if you don't want to share publicly.

**Comment by akhileshh** [comment]
Yes, I have emailed you a working link. The code to generate manifest is located [here](https://github.com/seung-lab/PyChunkedGraph/blob/5b38f5607b7dbd9c3d0abf4724097f97fa91c76f/pychunkedgraph/meshing/manifest/multiscale.py#L154).
